src/controllers/controllers/controller.py

```diff
@@ -153,33 +179,80 @@ def get_observation(self, policy):
 
         odom, lidar = self.get_data()
         odom = process_odom(odom)
-        if self.firstOdom:
-            self.offset = odom[0:6]
-            self.firstOdom = False
-        odom[0] = odom[0] - self.offset[0]
-        odom[1] = odom[1] - self.offset[1]
-        odom[2] = odom[2] - self.offset[2]
-        odom[3] = odom[3] - self.offset[3]
-        odom[4] = odom[4] - self.offset[4]
-        odom[5] = odom[5] - self.offset[5]
+
+        limited_odom = odom[-2:]
+        # if self.firstOdom:
```

Comment:
I know it looks ugly but this needs to stay for the real car odom to work. Can probably be changed to look nicer but logic is necessary.